src/App.js

```diff
@@ -2,15 +2,60 @@ import React from 'react';
 import './App.scss';
 
 class App extends React.Component {
-  addOne = () => {};
+  state = {
+    count: 0,
+  }
+
+  addOne = () => {
+    this.setState(state => ({
```

Comment:
Якщо передаєш в setState колбек, то краще явно вказувати, що посилання на попередній state
```suggestion
    this.setState(prevState => ({
```

Reply:
Міша у відео говорить, що там абсолютно достатньо називати state ----> відео "Functional setState" у темі "Updating state" -- 5:09 :)